redbadger/website-honestly
# Contact us service

### Motivation
![gif](http://i.giphy.com/mT47qMnQrl7LW.gif)
**this is the backend aspect of the cain slice on [the homepage design](https://app.zeplin.io/project.html#pid=57c8297fc3cf63795b6a4520&sid=57e003a43cd939017f48e3ae)**
[Trello ticket](https://trello.com/c/sHmEhkKG)

### Test plan
- ensure that when you hit the given endpoint with form data, an email will be sent to the emails in env, and returns a success response
- ensure that when you hit the given endpoint without form data, an email will not be sent and returns an appropriate response (that will trigger a user facing error message in the front end aspect of this work)
- ensure that the endpoint can be called through a browser (CORS are enabled)

- **ensure that if the endpoint falls over, we get notified**

### Pre-flight checklist
- [ ] Has been design reviewed - N/A
- [x] Add a gif - `![gif](gif.gif)`
